backend/api/Controllers/MissionController.cs

```diff
@@ -141,20 +216,114 @@ public async Task<ActionResult<byte[]>> GetMap([FromRoute] string assetCode, str
     }
 
     /// <summary>
-    /// Schedule a new mission
+    /// Reschedule an existing mission
```

Comment:
```suggestion
    /// Schedule an existing mission definition
```
Not sure I agree with naming this a reschedule. The mission definitions may have changed since the last time as well so better to call it schedule.